speech/setup.py

```diff
@@ -52,7 +52,6 @@
 REQUIREMENTS = [
     'google-cloud-core >= 0.20.0',
     'gapic-google-cloud-speech-v1beta1 >= 0.11.1, < 0.12.0',
-    'grpc-google-cloud-speech-v1beta1 >= 0.11.1, < 0.12.0',
```

Comment:
It's a dependency of `gapic-google-cloud-speech-v1beta1` so it gets installed automatically.